Source/Core/Common/Common.vcxproj

```diff
@@ -175,6 +176,7 @@
     <ClCompile Include="Hash.cpp" />
     <ClCompile Include="IniFile.cpp" />
     <ClCompile Include="JitRegister.cpp" />
+    <ClCompile Include="LdrWatcher.cpp" />
```

Comment:
_Windows-only_ CMake side i believe? Not gonna work on other OS. See build fails.